### DISCUSSION TITLE
Bot labelling me as previously graduated even tho I'm not

### DISCUSSION THREAD

**Discussion by vj-codes** [community/community#17764] [GitHub Education] [answered]
So last year, I had created a PR but my student developer pack wasn't approved by the deadline and couldn't graduate.
This year, I've got the pack, created the PR but bot claimed I already graduated.
Any roundabouts or whom shall I contact?

**Comment by elisemoe** [comment, marked as answer]
Hi @vj-codes 👋  we just updated our automation to account for PRs that didn't merge last year. Please open a new PR and let us know if you're still having trouble!

**Reply by vj-codes** [reply]
@elisemoe thanks for the suggestion tried doing that, still the bot works the same way and closed my PR.
here's it's link - https://github.com/education/GitHubGraduation-2022/pull/3058

**Reply by elisemoe** [reply]
Please add a link to your PR [in this Issue](https://github.com/education/GitHubGraduation-2022/issues/4290) and we'll manually review your Pull Request.
